package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "ceros-plugins",
+  "version": "1.0.0",
+  "description": "A collection of plugins and integrations for Ceros experiences",
+  "main": "Gruntfile.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Ah, yeah, I'll delete this. That's the default line that npm init puts in if you don't enter anything as the test command.